### DISCUSSION TITLE
Default subdivisionScheme CatClark not written to file

### DISCUSSION THREAD

**Discussion by yolov** [Autodesk/maya-usd#2205] [General]
Hi folks,

I decided to start a discussion to get a bit deeper into this before we (or someone) turns it into an issue.
When exporting from MayaUSD, the default subdivision scheme, which is Catmull-Clark is not written to the file at all. Only when the scheme is !default is it written to the file.
For example , the default doesn't leave anything in the file, while 'none' looks like this:
`uniform token subdivisionScheme = "none"`

![usd_subdscheme](https://user-images.githubusercontent.com/20951458/159301239-46ecaf79-42ac-4274-a10d-60fd73678a7f.png)

I guess it must be according to the standard, but this poses a few issues for us because it seems the rendering behavior is different between Maya and Houdini.
If I create a simple plane in Maya (just a plane, not subdivided, no nothing) - it will render as a plane.
If I export it to USD and bring it back in Maya - still renders as a plane.
If I bring it to Houdini with USD - it will render as a disc (because no token means cat-clark for Houdini as well).
And yes, there's the option in Houdini to manually override it to 'none', but I think it would be ideal to get the expected behavior with the least amount of clicks.

Alex

**Comment by santosg87** [comment]
hey Alex!

you bring an interesting situation here. 
currently, for Maya, we don't support smooth mesh display on USD data, so when we export that data, Maya will always just display the low complexity model.  

with that said, USdview displays the geometry the same as maya by default - on low complexity. 

there are a couple of ideas i see here to solve something like this:
1. we could have the default maya subdivision scheme set to "none". this would force a scheme option and the looks would be the same across DCCs given that there is no smooth mesh scheme. (this might cause annoyances later on when trying to render something in smooth mode. :) )
2. the other option we would have here is to fix the complexity support so a default smooth mesh can be set. This would probably enable Maya to look the same as, say, houdini - but it would still need an agreement across other DCC's on what should be the default. other places might still look different.


on a side note - while looking into our exporter, it seems we don't fully respect the interpolation boundary at export. I will log a bug for that. :)
